src/core/graphQL-interface/GraphQL.ts

```diff
@@ -177,6 +179,13 @@ function createResolvers(core: PerspectivismCore) {
                 console.log("runtimeMessageOutbox")
                 const { filter } = args
                 return core.runtimeService.getMessagesOutbox(filter)
+            },
+            //@ts-ignore
+            runtimeInfo: (parent, args) => {
+                return {
+                    ad4mExecutorVersion: ad4mExecutorVersion,
```

Comment:
Should we instead return this, since GraphQL/client users don't need to care the version of ad4m, it's encapsulated in ad4m-executor
```json
{
"name": "ad4m",
"version": "ad4m-executor-version"
}
```

Reply:
Yeah I think you are right actually, chances are the person using the ad4m-client will also know which version of ad4m they are using since they would have installed it